_posts/2023-06-06-Client-cert-auth-without-roles.adoc

```diff
@@ -0,0 +1,233 @@
+---
```

Comment:
Please change the file name to today's date (so the right date appears when we merge it).